frontend/src/actions/parcelsActions.tsx

```diff
@@ -22,48 +22,74 @@ export const storeParcelsAction = (parcelList: IProperty[]): IStoreParcelsAction
 //Parcel Detail API action
 export interface IAddress {
   line1: string;
-  line2: string;
-  city: string;
-  province: string;
+  line2?: string;
+  city?: string;
+  cityId: number | any;
+  province?: string;
+  provinceId: string;
   postal: string;
 }
 
 export interface IBuilding {
   id: number;
+  localId: string;
   description: string;
   address: IAddress;
-  latitude: number;
-  longitude: number;
-  localId: string;
-  constructionType: string;
-  postal: string;
-  buildingFloorCount: number;
-  buildingPredominateUse: string;
+  latitude: number | any;
+  longitude: number | any;
+  buildingFloorCount: number | any;
+  buildingConstructionType: string | undefined;
+  buildingConstructionTypeId: number | any;
+  buildingPredominateUse: string | undefined;
+  buildingPredominateUseId: string | any;
+  buildingOccupantType: string | undefined;
+  buildingOccupantTypeId: number | any;
+  leaseExpiry?: string;
+  occupantName: string;
+  transferLeaseOnSale: boolean;
   buildingTenancy: string;
+  buildingNetBookValue: number;
```

Comment:
Odd you have this value but none of the others related to evaluation?

Reply:
Hmm, I thought this came from the merge, I'll double check and make sure its being used.